components/base_page.js

```diff
@@ -11,10 +12,23 @@ import css from '../styles/common.module.scss';
 import utilsCss from '../styles/utils.module.scss';
 
 export default function Page({ children }) {
+    const router = useRouter();
+    const path = router.pathname;
+    console.log(path);
     return (
         <div className={css.container}>
             <Head>
-                <title>Awe and Reverence</title>
+                <title>
+                    {path
+                        .replace(/\//, '')
+                        .replace(
+                            /\w+/,
+                            path.charAt(1).toUpperCase() +
+                                path.substring(2, path.length) +
+                                ' | '
+                        )}
```

Comment:
*While you can, you probably shouldn't.*

Please define an intermediate variable, and just use it here, for better maintainability and readability.

Also, we should probably be defining a mapping of paths to page titles, rather than massaging the path part of the URL, because the URL was never meant to be an optimal human-readable title, but merely approximate.

For inspiration, see how we do this for Maskil in Django:

- https://github.com/aweandreverence/maskil-www/blob/1198bc89ade9acd4a878f0af094ce60708c397e6/maskil/constants/seo.py#L7-L33
- https://github.com/aweandreverence/django-htk/blob/4580b43941933f5b778ad4c423e3610c8718670d/view_helpers.py#L117-L140
- https://github.com/aweandreverence/django-htk/blob/4580b43941933f5b778ad4c423e3610c8718670d/view_helpers.py#L234-L249
- https://github.com/aweandreverence/django-htk/blob/4580b43941933f5b778ad4c423e3610c8718670d/view_helpers.py#L315-L330

Tangentially, for redirecting to the SEO-version of a URL, we should adapt the logic from `restful_obj_seo_redirect` to JS: https://github.com/aweandreverence/django-htk/blob/4580b43941933f5b778ad4c423e3610c8718670d/decorators/classes.py#L45-L69

Reply:
I made a new file in constants called page_titles.js. In it i have a list of key-value pairs, the key for the path, the value for the page title. In components/base_page.js, I used the object to identify the correct title depending on the file path. Combining it with ' | ' and the site title ('Awe & Reverence') produces the full title.